_includes/sitemap

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  {% include files-listed.html %}
```

Comment:
I put this here, and not at the top, because XML is very finicky about the XML declaration being on line 1 of the document. Putting this include at the top of the template adds a newline that invalidates the XML output.